.env.development

```diff
@@ -13,3 +13,5 @@ ACCESS_TOKEN_COOKIE_NAME='edx-jwt-cookie-header-payload'
 USER_INFO_COOKIE_NAME='edx-user-info'
 SEGMENT_KEY=''
 PORT=8734
+ALGOLIA_APP_ID=''
+ALGOLIA_SEARCH_API_KEY=''
```

Comment:
Add these config settings to edx-internal: https://github.com/edx/edx-internal/pull/2219